League_Api.Tests/League_ApiTests.cs

```diff
@@ -72,5 +73,15 @@ public async Task TestEmptyFunction()
             Assert.NotNull(response.Body);
             Assert.Equal(200, response.StatusCode);            
         }
+
+        [Fact]
+        public async Task TestPost()
+        {
```

Comment:
Try this bit of code
`            var function = new QuizRequestFunction();
            APIGatewayProxyRequest request = new APIGatewayProxyRequest();

            var body = new QuizRequestModel(2,3,2,3);

            request.Body = JsonConvert.SerializeObject(body);
            var context = new TestLambdaContext();
            var response = await function.Execute(request, context);`